### DISCUSSION TITLE
search engine restrictions

### DISCUSSION THREAD

**Discussion by sfregosi** [NMML/nfs-berpac#41] [Q&A] [answered]
Hi @jmlondon et al. This site is really awesome and inspirational! @dsjohnson shared with our PIFSC Openscapes Cohort and so I've been poking around. You mentioned that this is public, but you have some search engine restrictions in place that makes it so, in theory, only people you share the link with will find the page. Can you share how exactly you did that? 

I'm thinking I'd like to set something similar up for our underwater gliders, but I don't necessarily want their real-time location super easy for the world to access. 

Thanks!
Selene

**Comment by jmlondon** [comment, marked as answer]
Selene

It's actually pretty simple, take a look at [this line on the YAML header](https://github.com/NMML/nfs-berpac/blob/16a622864010010e5d55bce26675535dd4b13332/report.qmd#L8).

I pulled that tidbit from [the Quarto docs](https://quarto.org/docs/reference/formats/html.html#:~:text=Website-,search,-Setting%20this%20to). But, I haven't fully tested it to confirm whether it really does what I think it does. Until NMFS has wider access/licensing for GitHub Enterprise (which enables GitHub Pages for private/internal repositories), this is probably an ok/good enough solution for non-sensitive information that you want to share with a team/collaborators but don't want showing up on the front page of google.

**Reply by sfregosi** [reply]
Thanks! That *is* simple. I think you are right that it should work for what I need for non-sensitive info. Plus, I can have my lab mates test it by trying to find it and see how well it works :) 

I do hope NMFS is able to set up GitHub Enterprise because the other site I have that does have more sensitive info I wouldn't trust this for. That one I just am "deploying" as a folder of .htmls that can be viewed locally and will keep it that way for now.
